client/tests/integration/smartcontracts/create_nft_for_every_user_trigger/src/lib.rs

```diff
@@ -16,7 +16,8 @@ use iroha_wasm::{data_model::prelude::*, prelude::*};
 fn main() {
     iroha_wasm::info!("Executing trigger");
 
-    let accounts = FindAllAccounts.execute();
+    let accounts = FindAllAccounts.execute().dbg_unwrap();
```

Comment:
we don't allow unwrap in the code base. What do others think?

Reply:
It's an example of a smartcontract, and here it has no way to recover from this error. So why not?